src/actions/index.jsx

```diff
@@ -0,0 +1,103 @@
+
+import {getAllBreedsURL,getBreedImageURL} from '../utils/urls'
+
+export const SET_BREED ='SET_BREED'
+export const SET_BREEDS_DATA = 'SET_BREEDS_DATA' 
+export const SET_SUB_BREED ='SET_SUB_BREED'
+export const ADD_ITEM_SUB_BREED ='ADD_ITEM_SUB_BREED'
+export const REMOVE_ITEM_SUB_BREED ='REMOVE_ITEM_SUB_BREED'
+export const SET_SUB_BREED_IMAGE='SET_SUB_BREED_IMAGE'
+export const ADD_ITEM_FAVORITE ='ADD_ITEM_FAVORITE'
+export const REMOVE_ITEM_FAVORITE ='REMOVE_ITEM_FAVORITE'
+
+const setBreedData = payload  => ({type: SET_BREEDS_DATA, payload})
```

Comment:
aun que es menos typing, siempre usas parentesis tambien.
```suggestion
const setBreedData = (payload)  => ({type: SET_BREEDS_DATA, payload})
```